docs/release/release-tasks.md

```diff
@@ -163,6 +163,9 @@ From this point forward changes which should land in the release have to be cher
 2. Update the [milestone applier config](https://github.com/kubernetes/test-infra/blob/0b17ef5ffd6c7aa7d8ca1372d837acfb85f7bec6/config/prow/plugins.yaml#L371) accordingly (e.g. `release-1.4: v1.4` and `main: v1.5`)
    <br>Prior art: [cluster-api: update milestone applier config for v1.3](https://github.com/kubernetes/test-infra/pull/26631)
 
+3. Update the GitHub Actions to work with the new release version.
```

Comment:
Just a note that this is covered in line ~452 under tasks for the CI lead with the creation of the other CI jobs. 

Is the idea to make this part of the release lead tasks?

Reply:
@killianmuldoon I think this came up from the retrospective in the previous cycle and if I am not mistaken from you 😄 While you know better, the idea was to cover it so that we don't forget it in future releases and so you had a patch (9115) that was mentioned in this PR.  I was not aware it was already covered and don't see the point of making it part of RL tasks, so probably can close it.

Reply:
I think the idea from the retro - if I remember right - was to put this in a checklist or some other easier to follow format - maybe something like the release issue, but broken down for the CI team so it's easier to follow.

I definitely missed this task when it should have been done during the last release cycle - :facepalm: - and I think I probably added it to this file in the first place.